package.json

```diff
@@ -92,6 +92,7 @@
     "minimist": "^1.2.0",
     "mongoose": "^4.9.8",
     "morgan": "^1.8.1",
+    "ms": "^2.0.0",
```

Comment:
We already have the https://www.npmjs.com/package/parse-duration package included, no need to import this new package.